python/cudnn/gemm/cutedsl/grouped/unfused/api.py

```
@@ -109,24 +110,56 @@ def execute(
         self._is_supported = self._implementation._is_supported
         self._compiled_kernel = self._implementation._compiled_kernel
 
 
 _cache_of_GroupedGemmSm100Objects: dict[tuple, GroupedGemmSm100] = {}
 
+# Operand-metadata key -> the wrapper's derived result. Holds one entry per distinct
+# (operand metadata, config) the process sees, the same growth as the op cache above.
+_wrapper_memo: dict = {}
+
 
 def _stride_order(tensor: torch.Tensor) -> Tuple[int, ...]:
```

Ruff (0.16.1)

[error] Undefined name `torch`

(F821)